crates/js-sys/src/lib.rs

```diff
@@ -3390,7 +3390,7 @@ extern "C" {
     ///
     /// [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString)
     #[wasm_bindgen(method, js_name = toLocaleTimeString)]
-    pub fn to_locale_time_string(this: &Date, locale: &str) -> JsString;
+    pub fn to_locale_time_string(this: &Date, locale: &str, options: &JsValue) -> JsString;
```

Comment:
Instead we can add a new method called `to_local_time_string_with_options()` that fills this gap. Making this also not a breaking change.

Reply:
I can do that but I should note I did it this way to match `to_locale_date_string` which takes options. Since these are related functions it would make it inconsistent to have one have a separate fn for options and one not. (Another option would be to change bith to have a with_options, but that's also breaking)

I don't have strong feelings about this personally though so just lmk how you want to handle it. Thanks!

Reply:
I see. I was following the pattern from `web-sys` actually. But yeah, we should still stick with `*_with_options()` simply to avoid the breaking change.

Thank you for your attention on this!